packages/block-editor/src/components/border-radius-control/all-input-control.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __experimentalUnitControl as UnitControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getAllValue, isValuesMixed, isValuesDefined } from './utils';
+
+export default function AllInputControl( {
+	onChange = noop,
```

Comment:
Do we need to set this to a default of `noop`? It looks like we're only passing this prop down to `UnitControl` which also has its default of `noop`, so we might be able to drop it here and remove the lodash import?

Reply:
✅ Done.